packages/defichain-testing/src/utils.ts

```diff
@@ -0,0 +1,236 @@
+import BigNumber from 'bignumber.js'
```

Comment:
I think it's a good idea to refactor and separate all these into different categories, `account.ts`, `poolpair`, `rawtxn` and ... then have them re-exported at `index.ts` this keeps the cognitive load down when we occasionally revisit this module for enhancement in the future. 

> Code climate has a inspection that limit of 250 loc per file also, going to hit soon.